Sources/performance-worker/WorkerService.swift

```diff
@@ -0,0 +1,62 @@
+/*
+ * Copyright 2024, gRPC Authors All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import GRPCCore
+import NIOCore
+
+@available(macOS 13.0, iOS 16.0, watchOS 9.0, tvOS 16.0, *)
+struct WorkerService: Grpc_Testing_WorkerService.ServiceProtocol {
+  var testClient: GRPCClient? = nil
+  var testServer: GRPCServer? = nil
```

Comment:
This won't work when we add other RPCs because we need this type to be `Sendable`. This means we need to turn the `struct` into a `class` and protect this state using a lock. You can wrap the state up in a `struct State` and then use a `NIOLockedValueBox` to hold the state (you'll need to import `NIOConcurrencyHelpers`).